### PR TITLE
fix: Adding a way to handle tables and prevent Genny wrong answers (PTL-1156)

### PR DESCRIPTION
**Pull request by dbarros1979**
Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
Y

Have you checked all new or changed links?
Y

Is there anything else you would like us to know?
N

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

**Comment by dbarros1979** [comment]
@wjhendry Please see the changes I did as an example. 

I managed to add the command arguments list and an example call of any options without changing anything in the page. I did that using HTML comments, where I included texts between . 

Doing that we don't change any page and Genny managed to get better answers. If you approve this strategy, I'm happy to replicate to all commands and tables I find.

**Comment by wjhendry** [comment]
I see you are using comments - that looks good.

**Comment by wjhendry** [comment]
> I see you are using comments - that looks good.

But I get a runtime error when I run `npm run start`

**Comment by wjhendry** [comment]
> @wjhendry Please see the changes I did as an example.
> 
> I managed to add the command arguments list and an example call of any options without changing anything in the page. I did that using HTML comments, where I included texts between '< ! -- ' and ' -- >'.
> 
> Doing that we don't change any page and Genny managed to get better answers. If you approve this strategy, I'm happy to replicate to all commands and tables I find.

I'm happy with this approach if it gives better results from Genny

**Comment by dbarros1979** [comment]
good, thank @wjhendry! I will complete this page tables and then we can approve and merge it.

**Comment by dbarros1979** [comment]
just finished my changes @wjhendry. Please let me know if you are happy.

File: docs/05_operations/02_commands/01_server-commands.md
Comment (wjhendry) [inline docs/05_operations/02_commands/01_server-commands.md:1]: we have some commands that have tables, but which have not had comments added. Can you check this list:
killProcess
killServer
LogLevel
migrateDictionary
startProcess
startServer
DictionaryBuilder
I can do these tomorrow if you want.

Comment (wjhendry) [inline docs/05_operations/02_commands/01_server-commands.md:1]: Same list of tables to check:
killProcess
killServer
LogLevel
migrateDictionary
startProcess
startServer
DictionaryBuilder

**Review by wjhendry — COMMENTED** [review]
I can do add these if you want me to

File: docs/05_operations/02_commands/01_server-commands.md
Comment (dbarros1979) [reply docs/05_operations/02_commands/01_server-commands.md:1]: > Same list of tables to check: killProcess killServer LogLevel migrateDictionary startProcess startServer DictionaryBuilder

thanks @wjhendry. Appreciate if you could address them.

Comment (wjhendry) [reply docs/05_operations/02_commands/01_server-commands.md:1]: amanhã de manhã!

**Review by wjhendry — APPROVED** [review]

**Pull request merged**